packages/manager/cypress/e2e/core/account/account-logout.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { LOGIN_ROOT } from 'src/constants';
+import { interceptGetAccount } from 'support/intercepts/account';
+
+describe('Logout Test', () => {
+  beforeEach(() => {
+    cy.tag('purpose:syntheticTesting');
+  });
+
+  /*
+   * - Confirms that Cloud Manager log out functionality works as expected.
+   * - Confirms that the login application is up after account logout.
+   */
+  it('can logout the account and redirect to login endpoint', () => {
+    interceptGetAccount().as('getAccount');
+
+    cy.visitWithLogin('/account');
+    cy.wait('@getAccount');
+
+    // User can click Logout via user menu.
+    cy.findByTestId('nav-group-profile').click();
+    cy.findByTestId('menu-item-Log Out')
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
```

Comment:
nit: we could make this a little more idiomatic using the `userMenu` and `userMenuButton` UI helpers

```suggestion
    ui.userMenuButton
      .find()
      .click();

    ui.userMenu
      .find()
      .should('be.visible')
      .within(() => {
        cy.findByText('Log Out').should('be.visible').click();
      });
```